aws/resource_aws_ec2_client_vpn_endpoint.go

```diff
@@ -0,0 +1,323 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsEc2ClientVpnEndpoint() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEc2ClientVpnEndpointCreate,
+		Read:   resourceAwsEc2ClientVpnEndpointRead,
+		Delete: resourceAwsEc2ClientVpnEndpointDelete,
+		Update: resourceAwsEc2ClientVpnEndpointUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"client_cidr_block": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"dns_servers": {
```

Comment:
This attribute is not being set into the Terraform state in `resourceAwsEc2ClientVpnEndpointRead` so Terraform is unable to appropriately perform drift detection or import the attribute.

```
--- FAIL: TestAccAwsEc2ClientVpnEndpoint_withDNSServers (24.62s)
    testing.go:538: Step 2 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=3) {
         (string) (len=13) "dns_servers.#": (string) (len=1) "2",
         (string) (len=22) "dns_servers.2112610058": (string) (len=7) "8.8.4.4",
         (string) (len=22) "dns_servers.2609393598": (string) (len=7) "8.8.8.8"
        }
```

We'll need to call `d.Set()` with the correct information from the API or if its not possible to get, use `ImportStateVerifyIgnore: []string{"dns_servers"},` in the acceptance testing.

Reply:
I'm opting to ignore this attribute due to the fact that the Describe action doesn't include it.